docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -10,6 +10,22 @@ A replication factor of at least 3 is highly recommended for any M3DB deployment
 
 M3DB will do its best to distribute shards evenly among the availability zones while still taking each individual node's weight into account, but if some of the availability zones have less available hosts than others then each host in that zone will be responsible for more shards than hosts in the other zones and will thus be subjected to heavier load.
 
+
+### Dangers of Replication Factor of 1 or 2
+
+Running with RF=1 or RF=2 is not recommended for any use cases (testing or production), and in [the future][1107] such
```

Comment:
nit: i'd use two sentences here:

> Running with RF=1 or RF=2 is not recommended for any use cases (testing or production).
> [In the future][1107], such topologies may be rejected by M3DB entirely.

think it's worth putting a logged warning in?

Reply:
I'd rather save any code changes for when we fully address that issue, as we may prohibit it entirely.